template/module/hooks.py

```diff
@@ -44,3 +44,12 @@ def uninstall_hook(cr, registry):
         Database registry, using v7 api.
     """
     raise NotImplementedError
+
+
+def post_load():
+    """Loaded before any model or data has been initialized.
+
+    This is ok as the post-load hook is for server-wide
```

Comment:
@pedrobaeza 
Thanks for feedback.
Added just one useful case.
(Sorry, I don't know all useful cases)